index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```

Comment:
Considering we know the name of the client, shouldn't change this for now?

Reply:
Ok cool, but are we using MMF, are we using Musik Forandrer Liv - we need a decision from client team!

Reply:
Client team here! We decided on MMF

Reply:
Changes made as requested, thanks y'all